{"name": "test_login_fail[data0]", "status": "passed", "parameters": [{"name": "data", "value": "{'id': 1, 'title': '5位验证码', 'url': 'http://192.168.3.157:8069/dongwang-app/app/user/login', 'json': {'type': '1', 'flashToken': '', 'deviceId': 'aaa', 'device': '1', 'smsCode': '12312', 'phone': '113157121093'}, 'expected': {'status_code': 500, 'message': '验证码输入错误,请重新输入'}}"}], "start": 1614073226660, "stop": 1614073226698, "uuid": "52893557-e850-419c-b257-b3709de572eb", "historyId": "0714aa69308a5eb63ae144fb2b3d04e9", "testCaseId": "62fb4c4c1bc44f1f23ceb53be330b5b0", "fullName": "case.test_login.TestLogin#test_login_fail", "labels": [{"name": "story", "value": "错误验证码"}, {"name": "feature", "value": "登录模块"}, {"name": "parentSuite", "value": "case"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "SC-201907251023"}, {"name": "thread", "value": "5088-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "case.test_login"}]}